src/core/binding.c

```diff
@@ -1430,15 +1411,14 @@ QuicBindingDropBlockedSourcePorts(
 _IRQL_requires_max_(DISPATCH_LEVEL)
```

Comment:
```
//
// Looks up or creates a connection to handle a chain of datagrams.
// Returns TRUE if the datagrams were delivered, and FALSE if they should be
// dropped.
//
```

Should the comment above use the term packets as well?

Reply:
Yes